lib/twitter_client.rb

```diff
@@ -0,0 +1,12 @@
+class TwitterClient
```

Comment:
I love this pattern. Isolated logic, Plain ol' Ruby object. East to test, easy to maintain. :+1: 

I'd would like to see similar objects for your interactions with other services.